contracts/contracts/token/WOETH.sol

```diff
@@ -65,7 +85,7 @@ contract WOETH is ERC4626, Governable, Initializable {
          * WOETH has already seen transactions. But it is rather annoying in unit test
          * environment.
          */
-        oethCreditsHighres = _getOETHCredits();
```

Comment:
Still need the `oethCreditsHighres = _getOETHCredits();`

Reply:
uff thanks nice catch: https://github.com/OriginProtocol/origin-dollar/pull/2447/commits/b68269df644f05a6329c1ee260fd334733e972a0